name.py

```diff
@@ -0,0 +1,25 @@
+"""
+:param List of first and last names and optional number
+:returns integration of all names that can be
+"""
+def full_names(first_name, lest_name,min_length=0):
+    if min_length ==0:
+        names=[(first,last) for first in first_name for last in lest_name]
+    else:
+        names = [(first, last) for first in first_name for last in lest_name if len(first)+len(last)+1>=min_length]
```

Comment:
We encourage you to write one line codes, but it got a little too long than it should.
Try to simplify this loop, if needed split it into several lines.
Also, remember to use spaces around operators.

Reply:
( = )